webpack.config.js

```diff
@@ -3,7 +3,7 @@ const path = require("path");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
 const ExtractTextPlugin = require("extract-text-webpack-plugin");
 
-module.exports = {
+const widgetConfig = {
     entry: "./src/PullToRefresh/widget/PullToRefresh.ts",
     output: {
         path: path.resolve(__dirname, "dist/tmp"),
```

Comment:
Target `libraryTarget: "umd"` should be `amd` else the minifycation will break the client.